src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.cpp

```diff
@@ -122,11 +123,16 @@ LCAOrbitalBuilder::LCAOrbitalBuilder(ParticleSet& els, ParticleSet& ions, Commun
   aAttrib.add(cuspInfo, "cuspInfo");
   aAttrib.add(h5_path, "href");
   aAttrib.add(PBCImages, "PBCimages");
+  aAttrib.add(twist, "twist");
   aAttrib.put(cur);
 
   if (cur != NULL)
     aAttrib.put(cur);
 
+  if (std::abs(twist[0]-0.0) > 1e-6 && std::abs(twist[1]-0.0) > 1e-6 &&
```

Comment:
I saw the other place <1e-6. It is better to put >=1e-6 here. && should be ||.

Reply:
Done

Reply:
Better to #define the magic number as well